linera-execution/src/policy.rs

```diff
@@ -120,7 +120,6 @@ impl ResourceControlPolicy {
     }
 }
 
-#[cfg(with_testing)]
```

Comment:
This is potential a little controversial but I don't immediately a better option